packages/eslint-plugin/src/rules/member-ordering.ts

```diff
@@ -605,29 +651,36 @@ export default util.createRule<Options, MessageIds>({
 
       // Find first member which isn't correctly sorted
       members.forEach(member => {
+        // Note: Not all members have names
         const name = getMemberName(member, context.getSourceCode());
+        if (name === null) {
```

Comment:
A closer port to the behavior from before would be `if (!name) {`... but this is existing untested code anyway (despite the Codecov complaint) so I don't mind either way.